### PR TITLE
add zenodo badge

### LINKED ISSUE #151: add zenodo badge
I enabled Zenodo integration for core, api, and dashboard. My understanding is that we need to make new releases before the zenodo badges will be generated.

### PR DESCRIPTION
**Pull request by wholmgren**
not sure what to do about what's new right now.

closes #151

### CODE REVIEW

**Comment by alorenzo175** [comment]
Does it automatically update? or should we use https://zenodo.org/record/3473593#.XZfNwvdlCis ?

**Comment by wholmgren** [comment]
not sure what yours actually links to. the "cite all versions" box says to use https://doi.org/10.5281/zenodo.3473592

**Comment by alorenzo175** [comment]
mine links to the beta tagged version

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/SolarArbiter/solarforecastarbiter-core.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/SolarArbiter/solarforecastarbiter-core/alerts/)
 [![codecov](https://codecov.io/gh/solararbiter/solarforecastarbiter-core/branch/master/graph/badge.svg)](https://codecov.io/gh/solararbiter/solarforecastarbiter-core)
 [![Documentation Status](https://readthedocs.org/projects/solarforecastarbiter-core/badge/?version=latest)](https://solarforecastarbiter-core.readthedocs.io/en/latest/?badge=latest)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3473591.svg)](https://doi.org/10.5281/zenodo.3473591)
```
Comment (alorenzo175) [inline README.md]:
```suggestion
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3473592.svg)](https://doi.org/10.5281/zenodo.3473592)
``` 

lets try cite all

**Pull request merged**